pkg/connector/userinfo.go

```diff
@@ -10,12 +10,27 @@ import (
 	"maunium.net/go/mautrix/bridgev2"
 	"maunium.net/go/mautrix/bridgev2/networkid"
 
+	"go.mau.fi/mautrix-meta/pkg/messagix/socket"
 	"go.mau.fi/mautrix-meta/pkg/messagix/types"
 	"go.mau.fi/mautrix-meta/pkg/metaid"
 	"go.mau.fi/mautrix-meta/pkg/msgconv"
 )
 
 func (m *MetaClient) GetUserInfo(ctx context.Context, ghost *bridgev2.Ghost) (*bridgev2.UserInfo, error) {
+	if ghost.Name == "" {
+		contactID := metaid.ParseUserID(ghost.ID)
+		resp, err := m.Client.ExecuteTasks(&socket.GetContactsFullTask{
+			ContactID: contactID,
+		})
+		log := m.UserLogin.Log
+		if err != nil {
+			log.Trace().Any("ContactID", contactID).Any("err", err).Msg("GetContactsFullTask failed")
+			return nil, err
+		}
+		if len(resp.LSDeleteThenInsertContact) > 0 {
+			return m.wrapUserInfo(resp.LSDeleteThenInsertContact[0]), nil
+		}
+	}
 	return nil, fmt.Errorf("getting user info is not supported")
```

Comment:
This error is no longer accurate

```suggestion
	return nil, nil
```